system/HTTP/Response.php

```diff
@@ -1036,10 +1060,33 @@ protected function sendCookies()
 
 		foreach ($this->cookies as $params)
 		{
-			// PHP cannot unpack array with string keys
-			$params = array_values($params);
+			if (PHP_VERSION_ID < 70300)
+			{
+				// For PHP 7.2 we need to use the hacky method of setting SameSite in the path
+				if (isset($params['samesite']))
+				{
+					if (in_array(strtolower($params['samesite']), ['none', 'lax', 'strict']))
+					{
+						$params['path'] = '; samesite=' . $params['samesite'];
```

Comment:
I think you mean here `.=`

Reply:
Correct! Will add a better test for this as well.